usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -80,36 +80,36 @@ SELECT
   vw_award_search.product_or_service_description,
   vw_award_search.naics_code,
   vw_award_search.naics_description,
-  TREASURY_ACCT.treasury_account_identifiers as treasury_accounts
-  FROM vw_award_search
-  INNER JOIN awards a ON a.id = vw_award_search.award_id
-  LEFT JOIN
-  (SELECT
+  TREASURY_ACCT.treasury_accounts
+FROM vw_award_search
+INNER JOIN awards a ON (a.id = vw_award_search.award_id)
+LEFT JOIN (
+  SELECT
     recipient_hash,
     legal_business_name AS recipient_name,
     duns
-  FROM recipient_lookup AS rlv
-  ) recipient_lookup ON recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL
-  LEFT OUTER JOIN (
+  FROM
+    recipient_lookup AS rlv
+) recipient_lookup ON (recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL)
+LEFT JOIN (
   SELECT
     faba.award_id,
     JSONB_AGG(
-      JSONB_BUILD_OBJECT(
+      DISTINCT JSONB_BUILD_OBJECT(
```

Comment:
If you pull it over as an array of strings from Postgres then you still have to do some special handling since the Postgres values are stored in a CSV as strings. Now for TAS that isn't too much of a problem since you only have numbers that will make up a concatenated string. But for Federal Accounts where the `account_title` could have a wide variety of different characters, you would run into some difficulties making sure that you pick a unique character to concat on with Postgres. By using an array of JSONs we can leverage json.loads() to handle the parsing for us.

Reply:
it looks so perfect I like that so what I would do is fix it I will put a count title looking so perfect though and I will pick a unique Handler